pkg/proc/proc_test.go

```diff
@@ -960,6 +972,9 @@ func TestStacktraceGoroutine(t *testing.T) {
 }
 
 func TestKill(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
Also curious why this is being skipped on arm. Killing a process should be arch independent.